Makefile

```diff
@@ -24,7 +34,7 @@ run_frontend:
 	# @java -jar $(JAR) server gefconfig.yml
 
 run_backend:
-	(cd backend-docker && go run)
+	(cd backend-docker && go run main.go)
```

Comment:
As far as I understood, having a file name after "run" is commonly accepted way of doing it. Right?